pyspectre/core.py

```diff
@@ -75,33 +78,34 @@ def simulate( netlist_path: str, includes: Iterable[str] = None
 
     #ret = os.system(cmd)
     ret = run( cmd
-             , check          = False
-             , stdin          = DEVNULL
-             , stdout         = DEVNULL
-             , stderr         = DEVNULL
-             , capture_output = False
+             , check         =True
+             , stdin         =DEVNULL
+             , stdout        =True
+             , stderr        =True
+             , capture_output=False
              , ).returncode
 
     if ret != 0:
-        if log_path:
-            with open(log_path, 'r', encoding = 'utf-8') as log_handle:
+        if log_path is not None:
+            with open(log_path, 'r', encoding='utf-8') as log_handle:
                 print(log_handle.read())
         else:
-            print( 'log_path was not specified, '
-                 + 're-run the simulation with a log_path to view error details.' )
-
-        raise(IOError( errno.EIO, os.strerror(errno.EIO)
-                     , f'spectre returned with non-zero exit code: {ret}'
-                     , ))
-
+            print( '\n\n\n+++++++++++++++++++++++++++++'
+                   'log_path was not specified, '
+                 + f'Using Default log file name {log}\n'
+                   f'+++++++++++++++++++++++++++++++++')
+
+        #raise(IOError( errno.EIO, os.strerror(errno.EIO)
+        #             , f'spectre returned with non-zero exit code: {ret}'
+        #             , ))
```

Comment:
Same thing as above, sometimes a log file is not specified on purpouse, that functionality remain